_layouts/default.html

```diff
@@ -6,7 +13,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
 {% seo %}
-    <link rel="stylesheet" href="{{ "/assets/css/style.css?v=" | append: site.github.build_revision | relative_url }}">
+    <link rel="stylesheet" href="{{ "/assets/css/colors-ColorScheme.css?v=" | append: site.github.build_revision | relative_url | replace: "ColorScheme", ColorScheme }}">
```

Comment:
It's better place the replace filter at the beginning. Only thus, is possible avoid conflicts with the others

Reply:
Why is it better putting the replace before?

Reply:
Also thanks for helping me develop the fork it has been much appreciated

Reply:
> Why is it better putting the replace before?

mainly because filters are chained. 

Imagine that after process `relative_url`, the string contains `ColorScheme` (eg. `http://host/ColorScheme/assets/css/colors-ColorScheme.css` blah blah...). Then, with the replacement, the css file doesn't load, raising a 404 - Not Found 😉.